test/parse.test.js

```diff
@@ -18,6 +18,7 @@ describe('Parse', () => {
   it('moment-js like formatted dates', () => {
     global.console.warn = jest.genMockFunction()// moment.js '2018-4-1 1:1:1:22' will throw warn
     let d = '20130108'
+    let f = '[interpolated] mm dd'
```

Comment:
I think we have such tests in `display.test.js` L#140

Reply:
You are totally right. Will fix accordingly. Still finding my way around the tests 😅